### ISSUE TITLE
Update wiki to es and en

### ISSUE THREAD

**Issue by JeanPiffaut** [wavebeem/pkmn.help#165]
Hi, I love the site, it is incredibly useful. I came to this site through a Twich event because Illojuan uses it a lot.
The improvement to be made is to change the wiki to which it redirects to when you want to get extra information about Pokemon.
Currently it redirects to Wikidex to get information in Spanish. Currently a wiki that has already surpassed by far and that also works is pokemon.fandom.com. This wiki has versions in several languages so it may work to standardize a bit the wiki that uses the web in other languages as well.

**Comment by wavebeem** [comment]
Hmm, this is interesting. I'm not a huge fan of Fandom wikis, but if the content on Pokémon Wiki for Spanish is better than Wikidex, that's important.

I think I would be more likely to include a link to both Wikidex _and_ Pokémon Wiki, since some people might prefer one over the other. And I'm really not sure how people feel across other languages.

I could add some analytics to see which links are more popular, too. Thanks for the idea!

**Comment by BrianWhiting** [comment]
It would be nice to be able to choose which site is used for the link. For instance, I use pokemondb.net most of the time because the UI is more tailored to find info quickly for just the games. I also use serebii.net if I want to deep-dive into topics within the games (so much info there). I rarely use bulbapedia (especially on my mobile device due to the barrage of ads they have) since it has info about everything you could imagine (i.e. TCG, anime, etc.).

Adding a dropdown in the settings page would probably be the best place for something like this.

I might be able to implement this feature if I can find some time. I just can't seem to put the game down right now!

**Comment by wavebeem** [comment]
yeah, i forget how bad bulbapedia is because i use firefox + an adblocker on mobile... even on a modern phone bulbapedia slows to a crawl with ads enabled... it's actually one of the main reasons i made pkmn.help in the first place lol...

the main concern here is i need to be able to look up a pokemon by name easily by generating a URL... all the wikis i've linked to seem ok with that.

im definitely not sure what UI would be best here so far... i like the idea of picking your favorite site, but most people probably don't go digging in the settings menu to change that. there could be a `<details>` expand with all the other sites, or something...

i should use pokemondb and serebii more because they really have better designs lol.

i would love some help on this if you want to. and yeah, i can't believe i had the energy to add team type matchups this weekend instead of just playing pokemon the whole time lol (despite the numerous issue, the game is fun still...)
